bitcoin/src/blockdata/locktime/absolute.rs

```diff
@@ -179,8 +62,7 @@ impl fmt::UpperHex for PackedLockTime {
 /// ```
 #[allow(clippy::derive_ord_xor_partial_ord)]
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
-#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
-#[cfg_attr(feature = "serde", serde(crate = "actual_serde"))]
+#[derive(Ord)] // will be removed in next commit
```

Comment:
This never gets removed, claimed as part of `51f8b9a3 drop Ord on absolute::LockTime; add Ord to Transaction` but not done.

Reply:
Oops, sorry! I rebased this like 20 times and must have lost it.